web/packages/teleport/src/components/EventRangePicker/Custom/Custom.tsx

```diff
@@ -49,45 +49,39 @@ export const CustomRange = forwardRef<
     to: undefined,
   });
 
-  function handleDayClick(selectedDay: Date) {
-    // Don't let select date past today.
-    if (isAfter(selectedDay, endOfDay(new Date()))) {
+  // This handler runs on *every* click, but we only call your onChange
+  // once both ends of the range are picked.
+  const handleSelect = ({ to, from }: { to: Date; from: Date }) => {
+    // if both to and from in the state are undefined, this is our first time picking
+    // we should only use the "from" to denote the start of the range
+    if (!selectedRange.to && !selectedRange.from) {
+      setSelectedRange({
+        from: from && startOfDay(from),
+        to: undefined,
+      });
       return;
     }
 
-    // Don't do anything if `selected day` == `selected from`
-    if (selectedRange?.from && isSameDay(selectedRange.from, selectedDay)) {
-      return;
-    }
```

Comment:
we might need this to not throw this error? (errors when selecting same day) 
<img width="631" alt="image" src="https://github.com/user-attachments/assets/e10a5bab-d79e-41be-99aa-4e6d752ff331" />

Reply:
yeah good call. now i just let them select the same day actually since its only undefined if they select the same day (i only want to see logs from the 3rd or whtaever, for example)